### PR TITLE
provide fix for rvd.xml timeout setting not working

### PR DESCRIPTION
**Pull request by croufay**
git push --set-upstream origin issue#20

### CODE REVIEW

**Comment by deruelle** [comment]
Thanks @croufay, @otsakir can you please review with priority ?

**Comment by FerUy** [comment]
@otsakir @croufay @deruelle please bear in mind that as discussed in Slack, this PR was done based on a Restcomm-Connect release approximately 30 versions earlier than the one that worked-around issue https://github.com/RestComm/Restcomm-Connect/issues/2411, as for https://github.com/RestComm/Restcomm-Connect/pull/2438. So, if this is merged, there will be an inconsistency between RC and RVD to be fixed later.
Also, tests in my environment still aren't working with this work.

File: designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java [hunk 1]
```diff
@@ -214,6 +214,7 @@ public static RvdConfig loadRvdXmlConfig(String pathToXml) {
             XmlParser xml = new XmlParser(pathToXml);
             rvdConfig.setAllowedCorsOrigins(xml.getElementList("/rvd/corsWhitelist/origin"));
             rvdConfig.setUssdSupport(xml.getElementContent("/rvd/ussdSupport"));
+            rvdConfig.setExternalServiceTimeout(xml.getElementContent("/rvd/externalServiceTimeout"));
```
Comment (otsakir) [inline designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java]: @croufay , in the event that  #202 is indeed a valid problem (please check my latest comments there) `externalServiceTimeout` is already lazily initialized here:

https://github.com/RestComm/Restcomm-Connect/issues/2410#issuecomment-325275040

Are you sure it's empty ?

Comment (croufay) [reply designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java]: @otsakir, the value in the rvd.xml is always empty so we default to 5000. We I checked the code I noticed that we are not actually getting the value from rvd.xml so I added a patch for that.

Comment (otsakir) [reply designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java]: Sorry @croufay , i pasted wrong reference. Here is what i meant:
https://github.com/RestComm/visual-designer/blob/edabbdda4c0cbfdc286e6c913c2f737728f3a287/designer/src/main/java/org/restcomm/connect/rvd/RvdConfiguration.java#L292-L292

The getter does the job lazily.

**Comment by otsakir** [comment]
@croufay , from late investigations we did together it seems that RVD worked fine.  Shall i go ahead and close this ?

**Pull request closed without merging**